index.html

```diff
@@ -0,0 +1,134 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Полезный ИКИТ</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="css/style.css">
+</head>
+<body>
+    <header class="menu">
+        <div class="container">
+            <div class="menu__content">
+                <div class="menu__leading-content">
+                    <nav class="nav">
+                        <a href="#" class="menu__logo">
+                            <img class="ikit-logo" src="img/logo-ikit.png" alt="">
+                        </a>
+                        <ul class="link">
+                            <li class="nav__link active"><a href="#">Главная</a></li>
+                            <li class="nav__link"><a href="#">Информация</a></li>
+                            <li class="nav__link"><a href="#">Новости</a></li>
+                            <li class="nav__link"><a href="#">Контакты</a></li>
+                        </ul>
+                    </nav>
+                </div>
+                <div class="menu__trailing-content">
+                    <a class="profile">Алексей Корниенко</a>
+                    <img class="avatar" src="img/Avatar.png" alt="">
+                </div>
```

Comment:
этому блоку можно было придумать имя avatar
```suggestion
                <div class="menu__trailing-content avatar">
                    <a class="avatar__profile">Алексей Корниенко</a>
                    <img class="avatar__image" src="img/Avatar.png" alt="">
                </div>
```